2010-02-08  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/alpha/nptl/timer_settime.c: fix
	include path.

--- a/ports/sysdeps/unix/sysv/linux/alpha/nptl/timer_settime.c
+++ b/ports/sysdeps/unix/sysv/linux/alpha/nptl/timer_settime.c
@@ -1 +1 @@
-#include <nptl/sysdeps/unix/sysv/linux/x86_64/timer_gettime.c>
+#include <nptl/sysdeps/unix/sysv/linux/x86_64/timer_settime.c>
